app/services/pagarme/response_parser.rb

```diff
@@ -0,0 +1,8 @@
+class Pagarme::ResponseParser
+  def self.process(json_objects)
+    formated_json = "[#{json_objects.gsub("\n\u0000", ',')[0..-2]}]"
```

Comment:
I believe you could extract this to it's own method and give it a proper name to describe this operation.